day-cohort-70/Bangazon-API-team-3
# Feature/stores added models updated

## Ticket:

36


## What Changed:

Added full logic for Stores to exit int database

added stores model, adjusted products model to take store_id instead of customer_id

deleted duplicate fixtures

changed fixtures to accomodate only existing products


**Testing Steps:**

pull down both api, and client repos.

*******in api side immediatley reset your database with ./seed_data.sh*****

open bangazon. you should be able to see all products list when clicking products tab.

open stores tab. you should see all stores, click view products, this should create a dropdown of all the items in that store. this grid still needs to be stylized better, but theres some learning to do with how next.js handles layouts or whatever is giong on there, that i've broken off into a separate ticket.
